step1/hackatom-node.sh

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env bash
 set -e
 
-# This script is for running a simapp node. TODO add more details
+# This script is for running a local node, mostly for testing and demo purposes.
+# It creates a chain with one validator, called `ALICE`. The native token of
+# this chain is "stake", and `ALICE`, being the only existing account in the
+# chain at genesis, has lots of them.
 
-# Cleanup if there is any previously cloned cosmos-sdk repo
-rm -rf cosmos-sdk
+# Clone the SDK repo, we use the latest stable version: v0.44.
+if [[ ! -d cosmos-sdk ]]
+then
+    git clone https://github.com/cosmos/cosmos-sdk --depth 2 --branch master
```

Comment:
In your video, the git clone step was a bit slow. I added this, maybe it'll be faster